### PR TITLE
DDF-4338 added icon to visually identify external links in docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

adds a visual cue to links in documentation that will take users away from DDF documentation

#### Who is reviewing it? 
@austinsteffes @garrettfreibott @mcalcote @ahoffer @alexaabrd @vinamartin 

#### Select relevant component teams: 
@codice/docs 
#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@djblue
@lessarderic
@ricklarsen - Documentation
@shaundmorris
@vinamartin

#### How should this be tested?

- builds successfully
- icons added
#### What are the relevant tickets?
[DDF-4338](https://codice.atlassian.net/browse/DDF-4338)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_coreConcepts/attribute-stores.adoc [hunk 1]
```diff
@@ -11,10 +11,10 @@
 |Standard
 |Support Status
 
-|https://tools.ietf.org/html/rfc4510[Lightweight Directory Access Protocol (LDAP/LDAPS)]
+a|https://tools.ietf.org/html/rfc4510[Lightweight Directory Access Protocol (LDAP/LDAPS)] {external-link}
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_coreConcepts/attribute-stores.adoc]: Is this a stray `a`

File: distribution/docs/src/main/resources/content/_metadataAttributes/core-attributes-table.adoc [hunk 2]
```diff
@@ -55,7 +55,7 @@
 |[[_location]]location
 |The primary geospatial location of the resource.
 |Geometry
-|Valid Well Known Text (WKT) per http://www.opengeospatial.org/standards/wkt-crs +
+a|Valid Well Known Text (WKT) per http://www.opengeospatial.org/standards/wkt-crs {external-link} +
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_metadataAttributes/core-attributes-table.adoc]: I am thinking this `a` means something now....

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_metadataAttributes/core-attributes-table.adoc]: it's tells asciidoctor that there is asciidoctor formatted content in this cell

File: distribution/docs/src/main/resources/content/_coreConcepts/attribute-stores.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_coreConcepts/attribute-stores.adoc]: nope, it's for formatting asciidoctor content within a table cell

File: distribution/docs/src/main/resources/content/_transformers/atom-queryresp-xformer.adoc [hunk 3]
```diff
@@ -3,9 +3,9 @@
 :subtype: queryResponse
 :status: published
 :link: _atom_query_response_transformer
-:summary: Transforms a query response into an http://tools.ietf.org/html/rfc4287[Atom 1.0] feed.
+:summary: Transforms a query response into an \http://tools.ietf.org/html/rfc4287[Atom 1.0] feed.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_transformers/atom-queryresp-xformer.adoc]: This one probably doesn't need to be unlinked. fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by alexaabrd** [comment]
Hero successful 🎉 
Only thing im not sure was intentional:
image

**Pull request merged**